src/pages/home/report/ReportActionsView.js

```diff
@@ -129,7 +129,12 @@ class ReportActionsView extends React.Component {
                 this.scrollToListBottom();
             }
         });
-        updateLastReadActionID(this.props.reportID);
+
+        // Only mark as read if the report is open
+        if (!this.props.isDrawerOpen) {
+            updateLastReadActionID(this.props.reportID);
```

Comment:
Since this change is in a file that affects all platforms, can you update your tests in the description to ensure that the expected behavior still functions properly on web/desktop/mobile-web?